tests/api_tests/cuda/test_cuda_model_cache_management.py

```diff
@@ -0,0 +1,54 @@
+import copy
+import pprint
+from marqo.client import Client
+from marqo.errors import MarqoApiError, MarqoError, MarqoWebError
+import unittest
+from tests.marqo_test import MarqoTestCase
+from marqo import enums
+from unittest import mock
+from tests.utilities import allow_environments
+from tests.utilities import classwide_decorate
+
+@classwide_decorate(allow_environments, allowed_configurations=["CUDA_DIND_MARQO_OS"])
+class TestAddDocuments(MarqoTestCase):
+
+    # NOTE: The cuda should already have model loaded in the startup
+    def setUp(self) -> None:
+        self.client = Client(**self.client_settings)
+        self.index_name = "test_index"
+        self.MODEL = "ViT-L/14"
```

Comment:
Change this to the smaller clip model

Reply:
updated. also add structure test based on the comments from py-marqo.